Challenge2/src/agent.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { Finding, FindingSeverity, FindingType, HandleTransaction, ethers } from "forta-agent";
+import { createAddress } from "forta-agent-tools";
+import { TestTransactionEvent, MockEthersProvider } from "forta-agent-tools/lib/test";
+import { provideHandleTransaction } from "./agent";
+import { UNI_SWAP_EVENT_ABI, UNI_POOL_FUNCTIONS_ABI } from "./constants";
+import { getRealPoolAddress } from "./agent";
+
+const mockUniFactoryAddress = createAddress("0x01");
+const mockInitCodeHash = "0x0000000000000000000000000000000000000000000000000000000000000000";
```

Comment:
You could use [_ethers.constants.HashZero_](https://docs.ethers.org/v5/api/utils/constants/#constants-HashZero) so you don't have to manually write this out. 🙂

Reply:
I'm not sure this would work, because it wanted specifically 32 bytes, so I had to type out 64 zeros

Reply:
Maybe you tried it and it didn't work, but per the _ethers_ doc: "The Hash Zero, which is 32 bytes (64 nibbles) of zero." [Here](https://github.com/ethers-io/ethers.js/blob/master/packages/constants/src.ts/hashes.ts#L1)'s the implementation on github.

Were you possible getting it confused with `ethers.contants.AddressZero`, which is right above it?